gsa/src/web/pages/scanconfigs/editdialog.js

```diff
@@ -185,8 +185,13 @@ const EditScanConfigDialog = ({
               onChange={onValueChange}
             />
           </FormGroup>
-
-          {!configIsInUse && (
+          {configIsInUse ? (
+            <h4>
+              {_(
+                'The config or policy is now in use by a task or audit, therefore only name and comment can be modified.',
```

Comment:
Most users do not really know that configs and policies are technically the same and ideally they should not know it. So this message might be a little confusing. We should probably show different messages for configs and policies.

Reply:
We will also need a translation for the new notification.
```suggestion
                'The config or policy is currently in use by a task or audit, therefore only name and comment can be modified.',
```